lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```
@@ -8,17 +8,32 @@ instances:
     #
   - min_collection_interval: 60
 
-    ## @param url - string - required
+    ## @param url - string - DEPRECATED
     ## url of the website to get lighthouse metrics of
     ## ex: "https://www.datadoghq.com"
     #
     url: <URL_PLACEHOLDER>
```

Change for validation:
```suggestion
    ## @param url - string - optional
    ## url of the website to get lighthouse metrics of. This option is deprecated.
    ## ex: "https://www.datadoghq.com"
    #
    url: <URL_PLACEHOLDER>
```